src/pseudopeople/default_configuration.yaml

```diff
@@ -138,6 +159,9 @@ taxes_w2_and_1099:
     income:
```

Comment:
The docs have noising for "Income / Wages". Can you clarify that there is no concept of wages? ie everything is income?

Reply:
Will do.